app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -395,6 +396,14 @@ class AttendeeViewModel(
             })
     }
 
+    fun checkEmail(attendees: ArrayList<Attendee>): Boolean {
+        (0 until attendees.size).forEach {
```

Comment:
Why do you need the index of the list? This is not C

Reply:
Fixed